src/import/helm.ts

```diff
@@ -39,7 +40,15 @@ export class ImportHelm extends ImportBase {
     const tmpDir = pullHelmRepo(chartUrl, chartName, chartVersion);
 
     const chartYamlFilePath = path.join(tmpDir, this.chartName, CHART_YAML);
-    const contents = Yaml.load(chartYamlFilePath);
+    let contents: any[];
+
+    const isLocalChart = !chartUrl.startsWith('http') && !chartUrl.startsWith('oci://');
+    if (isLocalChart) {
+      const yamlContent = fs.readFileSync(chartYamlFilePath, 'utf-8');
```

Comment:
How come? Won't the existing `Yaml.load` work just as well?

Reply:
Yeah my bad, fixed in https://github.com/cdk8s-team/cdk8s-cli/pull/3667/commits/2cc505018c54601bee98dd7a8296ac78eb16af27.